src/main/java/co/cask/plugin/format/DBTypes.java

```diff
@@ -124,8 +132,14 @@
     return type;
   }
 
+  private static String formatDate(long time, String format) {
+		DateFormat dateFormat = new SimpleDateFormat(format);
+		return dateFormat.format(new Date(time));
+	}
```

Comment:
formatting is weird

Reply:
also, it will be good to call this method during `configure` stage through a `validate` method so we can catch syntax errors earlier.

Reply:
Fixed formatting. Added checks.